files/config/default/etc/workflows/uct-publish-fast.xml

```diff
@@ -0,0 +1,430 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<definition xmlns="http://workflow.opencastproject.org">
+
+  <id>fast</id>
+  <title>UCT - Publish fast</title>
```

Comment:
Let's make this more descriptive

  <title>UCT - Publish Opencast Studio upload</title>